x-pack/qa/smoke-test-index-lifecycle-with-security/build.gradle

```diff
@@ -0,0 +1,43 @@
+apply plugin: 'elasticsearch.standalone-rest-test'
+apply plugin: 'elasticsearch.rest-test'
+
+dependencies {
+  testCompile project(path: xpackProject('plugin').path, configuration: 'testArtifacts')
+}
+
+// bring in machine learning rest test suite
+task copyILMRestTests(type: Copy) {
+  into project.sourceSets.test.output.resourcesDir
+  from project(xpackProject('plugin').path).sourceSets.test.resources.srcDirs
```

Comment:
I realize we use this elsewhere, but was wondering if there's really a reason for it ? `xpackProject('plugin')` already returns a `Project` why wrap it in another call ? The additional `project(.path)` seems unnecessary.